src/app.py

```diff
@@ -12,9 +14,23 @@
 KEYS_DIR = "/keys"
 WORK_DIR = "/var/tmp"
 ROLES_DIR = os.environ.get("ROLES_DIR", "{}/roles".format(WORK_DIR))
+IMG_MINT_DIR = "{}/images".format(WORK_DIR)
 
 CONSTRUCTS = {}
 
+def execute_command(command):
+    process = subprocess.Popen(command.split(), stdout=subprocess.PIPE)
+    output, error = process.communicate()
+
+    if process.returncode > 0:
+        print('Executing command \"%s\" returned a non-zero status code %d' % (command, process.returncode))
+        sys.exit(process.returncode)
```

Comment:
don't `sys.exit` on failure (error message should be created to return as API request response)